package.json

```diff
@@ -72,6 +72,9 @@
     "verbose": true,
     "preset": "ts-jest/presets/js-with-ts",
     "testEnvironment": "jsdom",
+    "setupFiles": [
+      "./test/env.js"
```

Comment:
Note: This is a workaround for Next.js image config not having a default, should be resolved in the next version:
https://github.com/vercel/next.js/discussions/18373#discussioncomment-114212